.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-python.black-formatter"
```

Comment:
It is next to impossible to work with black without some formatter in VS Code...

Reply:
I think this also needs to run on save somehow. Or be a task in vs code.